Gum/Managers/HotkeyManager.cs

```diff
@@ -219,39 +219,21 @@ public static System.Windows.Forms.KeyEventArgs ConvertToFormsKeyEventArgs(Syste
 
     public void HandleKeyDownAppWide(KeyEventArgs e)
     {
-        // TODO: Move these values somewhere
-        double defaultValue = 8.25f;
-        double minValue = 6f;
-        double maxValue = 50f;
-
-        MainPanelControl tempMPC = GumCommands.Self.GuiCommands.mainPanelControl;
-        if (ZoomCameraIn.IsPressed(e) || ZoomCameraInAlternative.IsPressed(e))
-        {
-            tempMPC.FontSize = ClampValue(tempMPC.FontSize + 2, minValue, maxValue);
-            float scale = (float)(tempMPC.FontSize / defaultValue);
-            System.Diagnostics.Debug.WriteLine(scale);
-            ElementTreeViewManager.Self.UpdateTreeviewIconScale(scale);
-            e.Handled = true;
-        }
-        else if (ZoomCameraOut.IsPressed(e) || ZoomCameraOutAlternative.IsPressed(e))
+        ScaleAppFont();
+       
+        void ScaleAppFont()
```

Comment:
Is this a method tucked inside another method?  I looked it up, it's a Local Function.  Why have a function that's inside another one I do not understand that?

Reply:
Nothing else in the class uses these functions, it's just a personal preference that I changed out of habit to keep the class clean, but not at all necessary.